json-path/src/main/java/com/jayway/jsonpath/internal/function/JsonPathFunctionFactory.java

```diff
@@ -0,0 +1,24 @@
+package com.jayway.jsonpath.internal.function;
```

Comment:
This should probably be moved to a package within `com.jayway.jsonpath.spi`, along with the implementation class.